topology.go

```diff
@@ -156,7 +164,51 @@ func QueryTopology(dbpool *pgxpool.Pool, params TopologyOptions) ([]*models.Comp
 	// If ID is present, we do not apply any filters to the root component
 	results = applyStatusFilter(results, params.ID != "", params.Status...)
 
-	return results, nil
+	var res TopologyResponse
+	res.Components = results
+	populateTopologyResult(results, &res)
+
+	res.Teams, err = GetTeamNamesOfComponents(ctx, dbpool, res.componentIDs)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get teams of components: %w", err)
+	}
+
+	return &res, nil
+}
+
+func GetTeamNamesOfComponents(ctx context.Context, dbpool *pgxpool.Pool, componentIDs []string) ([]string, error) {
+	if len(componentIDs) == 0 {
+		return nil, nil
+	}
+
+	placeholders := make([]string, len(componentIDs))
+	for i := range componentIDs {
+		placeholders[i] = fmt.Sprintf("$%d", i+1)
+	}
+
+	args := make([]any, len(componentIDs))
+	for i, id := range componentIDs {
+		args[i] = id
+	}
+
+	query := fmt.Sprintf("SELECT DISTINCT teams.name FROM team_components LEFT JOIN teams ON teams.id = team_components.team_id WHERE component_id IN (%s)", strings.Join(placeholders, ","))
```

Comment:
Try to make this part of the Topology Query, not a separate query

Reply:
Leavining this as it is for now since we'll be moving to topology view.

Reply:
@adityathebe its better to remove this as discussed in yesterday's call

Fixing the topology view should be the priority

Reply:
I must have misheard but I thought that wasn't the priority right now - adding metadata was.

Reply:
Yeah, basically, skip teams for now, add other metadata and then do topology view with teams